packages/portal-proto/tailwind.config.js

```diff
@@ -783,7 +783,7 @@ module.exports = {
           borderAlt: "#F0E4CB",
         },
         warningColor: "#FFAD0D", //orangish
-        warningColorText: "#976F21", //lighter orangish
+        warningColorText: "#8A661E", //lighter orangish
```

Comment:
This was changed for accessibility contrast reasons.

Reply:
This is fine